From 702e9295b718b5f516aa8e3989af2b52048773a1 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Thu, 23 Jan 2020 19:19:24 +1300
Subject: remove non-unix deps

---
 Cargo.toml | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index d1d1c70..f74654e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -186,18 +186,3 @@ features = ["derive"]
 version = "0.2.0"
 features = ["macros"]
 default-features = false
-[target."cfg(target_arch = \"wasm32\")".dependencies.js-sys]
-version = "0.3.28"
-
-[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen]
-version = "0.2.51"
-features = ["serde-serialize"]
-
-[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen-futures]
-version = "0.4.1"
-
-[target."cfg(target_arch = \"wasm32\")".dependencies.web-sys]
-version = "0.3.25"
-features = ["Headers", "Request", "RequestInit", "Response", "Window"]
-[target."cfg(windows)".dependencies.winreg]
-version = "0.6"
-- 
2.24.0

