From acf2465eca0498220071d775e0df4ce685776668 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Tue, 19 Nov 2019 12:45:00 +1300
Subject: Remove non-unix dependencies

---
 Cargo.toml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index fef331e..472b5e5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,9 +23,6 @@ repository = "https://github.com/danburkert/memmap-rs"
 version = "0.3"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["basetsd", "handleapi", "memoryapi", "minwindef", "std", "sysinfoapi"]
 [badges.appveyor]
 repository = "danburkert/mmap"
 
-- 
2.23.0

