diff --git a/src/libssl/ssl.rs b/src/libssl/ssl.rs
index f5e0080..9eb905b 100644
--- a/src/libssl/ssl.rs
+++ b/src/libssl/ssl.rs
@@ -3033,6 +3033,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(feature = "network_tests")]
     fn ssl_on_nonblocking_socket() {
         let sock = net::TcpStream::connect("mesalink.io:443").expect("Conenction failed");
         assert_eq!(true, sock.set_nonblocking(true).is_ok());
@@ -3375,6 +3376,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(feature = "network_tests")]
     fn get_ssl_fd() {
         let ctx_ptr = mesalink_SSL_CTX_new(mesalink_TLS_client_method());
         let ssl_ptr = mesalink_SSL_new(ctx_ptr);
@@ -3525,6 +3527,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(feature = "network_tests")]
     fn early_data_to_mesalink_io() {
         const HTTP_REQUEST: &[u8; 83] = b"GET / HTTP/1.1\r\n\
             Host: mesalink.io\r\n\
