diff --git a/Cargo.toml b/Cargo.toml
index 474c4e3..22e8335 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,10 +38,6 @@ overflow-checks = true
 name = "mesalink"
 crate-type = ["lib"]
 
-[[example]]
-name = "bogo_shim"
-path = "examples/bogo_shim/bogo_shim.rs"
-required-features = ["verifier", "error_strings", "tls13", "aesgcm", "chachapoly", "x25519", "ecdh", "ecdsa"]
 [dependencies.base64]
 version = "0.10"
 
