From f80b0fa9536a55aaa1917d2f3b00475925f8791c Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Thu, 7 Nov 2019 13:36:07 +1300
Subject: Remove non-unix or nightly-only dependencies

---
 Cargo.toml | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 1c1f718..56af4f4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,38 +24,17 @@ repository = "https://github.com/rust-random/getrandom"
 [dependencies.cfg-if]
 version = "0.1.2"
 
-[dependencies.compiler_builtins]
-version = "0.1"
-optional = true
-
-[dependencies.core]
-version = "1.0"
-optional = true
-package = "rustc-std-workspace-core"
-
 [dependencies.log]
 version = "0.4"
 optional = true
 
 [features]
 dummy = []
-rustc-dep-of-std = ["compiler_builtins", "core"]
 std = []
-test-in-browser = ["wasm-bindgen"]
-[target."cfg(target_os = \"wasi\")".dependencies.wasi]
-version = "0.9"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.64"
 default-features = false
-[target.wasm32-unknown-unknown.dependencies.stdweb]
-version = "0.4.18"
-optional = true
 
-[target.wasm32-unknown-unknown.dependencies.wasm-bindgen]
-version = "0.2.29"
-optional = true
-[target.wasm32-unknown-unknown.dev-dependencies.wasm-bindgen-test]
-version = "0.2"
 [badges.appveyor]
 repository = "rust-random/getrandom"
 
-- 
2.25.0

