From b09cb66e871e583a89b0e9a8b6dce6e28680e089 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Tue, 3 Dec 2019 20:25:24 +1300
Subject: Disable broken dot graph test

---
 tests/graph.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/graph.rs b/tests/graph.rs
index 773a993..92f9eb2 100644
--- a/tests/graph.rs
+++ b/tests/graph.rs
@@ -1411,6 +1411,7 @@ fn neighbor_order() {
                vec![b, c, c, a]);
 }
 
+#[cfg(broken_tests)]
 #[test]
 fn dot() {
     // test alternate formatting
-- 
2.24.0

