From a974e90344f4b752cd1c5c370962336cad61bdf5 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Wed, 13 Nov 2019 17:11:44 +1300
Subject: Remove non-unix deps

---
 Cargo.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 403d50c..1eaf253 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,5 +60,3 @@ version = "0.1"
 version = "0.2"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.50"
-[target."cfg(windows)".dependencies.getrandom]
-version = "0.1.12"
-- 
2.23.0

