From 1f1d4a5e072173da9107c6413ca7f6090a56f3e6 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Fri, 27 Dec 2019 20:41:39 +1300
Subject: Handle failure

Bug: https://github.com/sfackler/rust-openssl/issues/1215
Bug: https://github.com/sfackler/rust-openssl/pull/1216
---
 src/ssl/test/mod.rs | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/ssl/test/mod.rs b/src/ssl/test/mod.rs
index 18cbb39..0f444aa 100644
--- a/src/ssl/test/mod.rs
+++ b/src/ssl/test/mod.rs
@@ -601,7 +601,10 @@ fn default_verify_paths() {
     ctx.set_default_verify_paths().unwrap();
     ctx.set_verify(SslVerifyMode::PEER);
     let ctx = ctx.build();
-    let s = TcpStream::connect("google.com:443").unwrap();
+    let s = match TcpStream::connect("google.com:443") {
+        Ok(s) => s,
+        Err(e) => return,
+    };
     let mut ssl = Ssl::new(&ctx).unwrap();
     ssl.set_hostname("google.com").unwrap();
     let mut socket = ssl.connect(s).unwrap();
-- 
2.24.0

