From aaead631b966b26046a5e760ce0096a5b405c365 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Fri, 3 Jan 2020 08:20:12 +1300
Subject: Remove stripped test mention in Cargo.toml

---
 Cargo.toml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 7e8f0b4..186464f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,8 +25,3 @@ rand = "0.3.5"
 
 name = "bench"
 path = "bench/bench.rs"
-
-[[test]]
-
-name = "test"
-path = "test/test.rs"
-- 
2.24.0

