From 8fbe7f93f455264a4879e39789cab7340478b778 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Thu, 14 Nov 2019 21:18:40 +1300
Subject: Disable roundtrip tests unless a cfg option is passed

Bug: https://github.com/Frommi/miniz_oxide/issues/64
---
 tests/test.rs | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/tests/test.rs b/tests/test.rs
index ea62a69..c299f89 100644
--- a/tests/test.rs
+++ b/tests/test.rs
@@ -2,7 +2,9 @@ extern crate miniz_oxide;
 
 use std::io::Read;
 
+#[cfg(roundtrip)]
 use miniz_oxide::deflate::compress_to_vec;
+
 use miniz_oxide::inflate::{decompress_to_vec, decompress_to_vec_zlib, TINFLStatus};
 
 fn get_test_file_data(name: &str) -> Vec<u8> {
@@ -47,12 +49,14 @@ fn decompress_zero_code_len_2() {
     let _ = decompress_to_vec(data.as_slice());
 }
 
+#[cfg(roundtrip)]
 fn get_test_data() -> Vec<u8> {
     use std::env;
     let path = env::var("TEST_FILE").unwrap_or_else(|_| "../miniz/miniz.c".to_string());
     get_test_file_data(&path)
 }
 
+#[cfg(roundtrip)]
 fn roundtrip(level: u8) {
     let data = get_test_data();
     let enc = compress_to_vec(&data.as_slice()[..], level);
@@ -66,16 +70,19 @@ fn roundtrip(level: u8) {
     assert!(data == dec);
 }
 
+#[cfg(roundtrip)]
 #[test]
 fn roundtrip_lvl_9() {
     roundtrip(9);
 }
 
+#[cfg(roundtrip)]
 #[test]
 fn roundtrip_lvl_1() {
     roundtrip(1);
 }
 
+#[cfg(roundtrip)]
 #[test]
 fn roundtrip_lvl_0() {
     roundtrip(0);
-- 
2.23.0

